node/node.d.ts

```diff
@@ -1371,7 +1371,8 @@ declare module "dgram" {
 
     interface Socket extends events.EventEmitter {
         send(buf: Buffer, offset: number, length: number, port: number, address: string, callback?: (error: Error, bytes: number) => void): void;
-        bind(port: number, address?: string, callback?: () => void): void;
+        send(buf: Buffer, port: number, address: string, callback?: (error: Error, bytes: number) => void): void;
```

Comment:
This overload should be the first in the list of `send` overloads, to help with resolution.